public/script.js

```diff
@@ -0,0 +1,249 @@
+    let element = objectToHtmlElement(playlist);
+    element.insertAdjacentHTML('beforeend', `<p>${playlist.description}</p>`);
+    playlistSection.querySelector('.items').insertAdjacentElement('beforeend', element);
+  });
+  return playlistSection;
+}
+
+/**
+* Вспомогальная функция для преобразования массива с именами исполнителей в строку
+* @param artists - массив с именами исполнителей
+* @returns строку с именами исполнителей 
+*/
+function artistsToString(artists) {
+  let result = '';
+  artists.forEach(function(artist, index) {
+    result += artist.name;
+    if(artists[index + 1])
+      result += ', ';
+  }
+  );
+  return result;
+}
+
+/**
+*  Функция для создания раздела с альбомами
+* @param {object[]} albums - массив альбомов
+* @returns раздел, содержащий ссылки на альбомы
+*/
+function albumsToHtmlElement(albums) {
+  let albumSection = createSection().cloneNode(true);
+  albumSection.querySelector('.section__header').textContent = 'Альбомы';
+  albums.forEach(album => {
+    let element = objectToHtmlElement(album);
+    element.insertAdjacentHTML('beforeend', `<p>${artistsToString(album.artists)}</p>`);
+    albumSection.querySelector('.items').insertAdjacentElement('beforeend', element);
+  });
+  return albumSection;
+}
+
+/**
+*  Функция для создания раздела с исполнителями
+* @param {object[]} artists - массив исполнителей
+* @returns раздел, содержащий ссылки на исполнителей
+*/
+function artistsToHtmlElement(artists) {
+  let artistSection = createSection().cloneNode(true);
+  artistSection.querySelector('.section__header').textContent = 'Исполнители';
+  artists.forEach(artist => {
+    artistSection.querySelector('.items').insertAdjacentElement('beforeend', objectToHtmlElement(artist));
+  });
+  return artistSection;
+}
+
+/**
+*  Функция для создания раздела с треками
+* @param {object[]} tracks - массив треков
+* @returns раздел, содержащий ссылки на треки
+*/
+function tracksToHtml(tracks) {
+  let trackSection = createSection().cloneNode(true);
+  trackSection.querySelector('.section__header').textContent = 'Треки';
+  trackSection.className = 'track-section';
+  trackSection.querySelector('.items').className = 'tracks';
+  tracks.forEach(track => {
+    trackSection.querySelector('.tracks').insertAdjacentHTML('beforeend', 
+    `<a href='${track.external_urls.spotify}' 
+     class='track link'>${artistsToString(track.artists)} - ${track.name}</a>`);
+  });
+  return trackSection;
+}
+
+/** 
+ * Функция для вывода результатов поиска на страницу 
+ * @param {object} obj - объект, содержащий результаты поиска
+*/
+function searchResults(obj) {
+  content.innerHTML = "";
+  const albums = obj.albums.items;
+  const artists = obj.artists.items;
+  const tracks = obj.tracks.items;
+  if(albums.length == 0 && artists.length == 0 && tracks.length == 0)
+    content.insertAdjacentHTML('afterbegin', '<div class="not-found">По вашему запросу ничего не найдено</div>');
+  else {
+    if(albums.length != 0)
```

Comment:
Можно написать так
```suggestion
    if(albums)
```
Также тут стоило бы использование сравнение без приведения типа
```suggestion
    if(albums.length !== 0)
```